x-pack/plugins/observability/public/pages/slo_details/components/events_chart_panel.tsx

```diff
@@ -62,7 +62,9 @@ export function EventsChartPanel({ slo, range }: Props) {
   const sloIndicator = cloneDeep(slo.indicator);
   if (instanceIdFilter) {
     sloIndicator.params.filter =
-      !!sloIndicator.params.filter && sloIndicator.params.filter.length > 0
+      !!sloIndicator.params.filter &&
+      typeof sloIndicator.params.filter === 'string' &&
```

Comment:
Should the events chart apply filters regardless of whether they are a string or unified search?

Reply:
i will modify the query.